starter-code/src/viking.js

```diff
@@ -1,11 +1,88 @@
 // Soldier
-function Soldier () {}
-
+function Soldier (health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+Soldier.prototype.attack = function (){
+  return  this.strength ;
+};
+Soldier.prototype.receiveDamage = function (damage){
+  this.health = this.health - damage;
+};
 // Viking
-function Viking () {}
+function Viking (name, health, strength) {
+  Soldier.call(this, health, strength);
+  this.name = name;
+}
+
+
+Viking.prototype = Object.create(Soldier.prototype) ;
+
+Viking.prototype.receiveDamage = function (damage){
+  this.health = this.health - damage;
+  if(this.health > 0){
+    return this.name + " has received " + damage + " points of damage";
+  } else {
+    return this.name + " has died in act of combat";
+  }
+};
+
+Viking.prototype.battleCry = function (){
+  return "Odin Owns You All!" ;
+};
 
 // Saxon
-function Saxon () {}
+function Saxon (health, strength) {
+  Soldier.call(health, strength);
```

Comment:
Aquí creo que se te ha olvidado como primer parámetro this. En el constructor de Vikings si lo tienes. Te lo recuerdo, porque es un argumento que debemos pasar, ya que representa el contexto en el que se encuentra nuestra función y en ocasiones este puede ser distinto al de la función desde la que llamamos al prototype del padre. (Esto lo veremos más adelante, pero para que te suene...) 
En este caso el contexto desde el que la llamamos es el de la clase Saxon por eso funciona bien.